src/plugins/blocksmith/utils/selection.js

```diff
@@ -0,0 +1,43 @@
+let windowSelection = {};
```

Comment:
@efjacobson - were you able to test this?

Reply:
@arunkarnati like, in a unit test? no. but if you remove the `selection.capture` and `selection.restore` calls in blocksmith you will see that when you command+c to copy you lose the selection

Reply:
@efjacobson - no, copying and pasting in the UI.